src/iv/imageviewer.cpp

```diff
@@ -386,6 +386,17 @@ ImageViewer::createActions()
     //    toggleImageAct->setEnabled(true);
     connect(toggleImageAct, SIGNAL(triggered()), this, SLOT(toggleImage()));
 
+    showDataWindowAct = new QAction(tr("Show Data Window"), this);
```

Comment:
Can I suggest that the menu text itself say "Show Display Window (dotted line)" and "Show Data Window (solid line)" to make it self-documenting.

Reply:
Surely do!